src/2_RTB/AnalisiDeiRequisiti/content.typ

```diff
@@ -408,6 +409,150 @@ UC8 : - Archiviazione delle anomalie rilevate
   - *Estensioni*: se il sistema non riesce a riconoscere o integrare correttamente il nuovo sensore, l'utente riceve una notifica di errore.
   */
 /* Bisogna capire se tenere la UC14 */
+
+/ UC13 : - Inserimento dati
+  - *Attore Principale*: Simulatore.
+  - *Precondizioni*: il simulatore è stato attivato e si è connesso al sistema.
+  - *Postcondizioni*: il simulatore è connesso al sistema ed ha immesso i dati.
+  - *Scenario Principale*:
+    - Il simulatore:
+      + si connette al sistema;
+      + avvia la simulazione di un particolare tipo di dato;
+      + inserisce il #glossary("dato sintetico") all'interno del sistema.
+  - *Specializzazioni*:
+    - Inserimento dati temperatura [UC13.1];
+    - Inserimento dati precipitazioni [UC13.2];
+    - Inserimento dati umidità [UC13.3];
+    - Inserimento dati #glossary("stato del cielo") [UC13.4];
+    - Inserimento dati vento [UC13.5];
+    - Inserimento dati inquinamento dell'aria [UC13.6];
+    - Inserimento dati indice UV [UC13.7];
+    - Inserimento dati posizione biciclette elettriche [UC13.8];
+    - Inserimento dati congestione stradale [UC13.9];
+    - Inserimento dati rumore [UC13.10];
+    - Inserimento dati scosse sismiche [UC13.11];
+    - Inserimento dati connessione alla Wi-Fi pubblica [UC13.12];
```

Comment:
"rete Wi-Fi pubblica"?

Reply:
ipotizzando di avere una wifi pubblica della città, avere un live counter sul numero di accessi ad essa per cella controllata. Mi sembrava figo, ma è un'idea.

Reply:
Sisi non metto in discussione l'idea, e' solo che qui l'hai chiamato  - Inserimento dati connessione alla Wi-Fi pubblica [UC13.12]; e sotto [quando esplichi lo UC vero e proprio] - Inserimento dati connessione Wi-Fi pubblica [UC13.12]; e i due dovrebbero coincidere tutto qua.